src/daemon/database/interface.rs

```diff
@@ -320,6 +324,31 @@ pub fn db_spending_vaults(
     )
 }
 
+/// Get the vaults that are in the process of being canceled, along with the respective Unvault
+/// transaction.
+pub fn db_canceling_vaults(
+    db_path: &PathBuf,
+) -> Result<Vec<(DbVault, UnvaultTransaction)>, DatabaseError> {
```

Comment:
Why are you getting the Unvault txs for? I think you want to express in SQL "hey gimme all the Cancel transactions for which the vault is marked as being `canceling`" so that you can use their txid to ask `bitcoind`: "is this transaction from my wallet confirmed?"